map.js

```diff
@@ -3,6 +3,9 @@ var DISTRICTS_CSV = "data/neighborhood_district.csv";
 
 var CENSUS_TRACT_COL = "CTract";
 
+//Neighborhood.csv uses a different column header name for census tract
```

Comment:
Normally we put an empty space after the '//'
Not important at all, so don't worry about changing it now, just for future code.